doc/dev-reference/getting_started.rst

```diff
@@ -41,9 +41,9 @@ Very Quick Start
      * ``MsBuild.exe`` --- use Visual Studio MsBuild
```

Comment:
Since MS Windows support has been completely removed since a few releases ago, I would also remove mentions to Windows builds.